Sources/Models/RequestModel.swift

```diff
@@ -37,4 +82,106 @@ open class RequestModel: Codable {
         responseHeaders = responseHttp.allHeaderFields as? [String: String]
     }
     
+    var curlRequest: String {
+        var components = ["$ curl -v"]
+
+        guard
+            let _ = self.host
+        else {
+            return "$ curl command could not be created"
+        }
+
+        if method != "GET" {
+            components.append("-X \(method)")
+        }
+
+        components += headers.map {
+            let escapedValue = String(describing: $0.value).replacingOccurrences(of: "\"", with: "\\\"")
+
+            return "-H \"\($0.key): \(escapedValue)\""
+        }
+
+        if let httpBodyData = httpBody, let httpBody = String(data: httpBodyData, encoding: .utf8) {
+            var escapedBody = httpBody.replacingOccurrences(of: "\\\"", with: "\\\\\"")
+            escapedBody = escapedBody.replacingOccurrences(of: "\"", with: "\\\"")
+
+            components.append("-d \"\(escapedBody)\"")
+        }
+        
+        for credential in credentials {
+            components.append("-u \(credential.0):\(credential.1)")
+        }
+        
+        if let cookies = cookies {
+            components.append("-b \"\(cookies[..<cookies.index(before: cookies.endIndex)])\"")
+        }
+
+        components.append("\"\(url)\"")
+
+        return components.joined(separator: " \\\n\t")
+    }
+    
+    var postmanItem: PMItem? {
+        guard
+            let url = URL(string: self.url),
+            let scheme = self.scheme,
+            let host = self.host
+            else { return nil }
+        
+        let dateFormatterGet = DateFormatter()
+        dateFormatterGet.dateFormat = "yyyyMMdd_HHmmss"
+        
+        let name = "\(dateFormatterGet.string(from: date))-\(url)"
+        
+        var headers: [PMHeader] = []
+        let method = self.method
+        for header in self.headers {
+            headers.append(PMHeader(key: header.0, value: header.1))
+        }
+        
+        let rawBody: String
+        if let httpBodyData = httpBody, let httpBody = String(data: httpBodyData, encoding: .utf8) {
+            rawBody = httpBody
+        }
+        else {
+            rawBody = ""
+        }
```

Comment:
I would remove this `else`, and assign the default value to the variable `rawBody`.